demisto_sdk/commands/find_dependencies/tests/find_dependencies_test.py

```diff
@@ -1,24 +1,294 @@
 import json
 import os
+from pathlib import Path
+from typing import List
 
+import demisto_sdk.commands.create_id_set.create_id_set as cis
 import networkx as nx
 import pytest
-from demisto_sdk.commands.common.legacy_git_tools import git_path
+from demisto_sdk.commands.common.constants import FileType
 from demisto_sdk.commands.find_dependencies.find_dependencies import \
     PackDependencies
+from TestSuite.repo import Repo
+from TestSuite.test_tools import ChangeCWD
 from TestSuite.utils import IsEqualFunctions
 
 
+def create_a_pack_entity(pack, entity_type: FileType = None, entity_id: str = None, entity_name: str = None,
+                         commands: List[str] = None):
+    """
+    Given
+        - A Pack.
+
+    When
+        - add an entity to the pack.
+
+    Then
+        - Adds the entity to the pack with basic data.
+    """
+    if entity_type == FileType.SCRIPT:
+        pack.create_script(entity_id).create_default_script(entity_id)
+    elif entity_type == FileType.INTEGRATION:
+        pack.create_integration(entity_id).create_default_integration(entity_id, commands)
+    elif entity_type == FileType.PLAYBOOK:
+        pack.create_playbook(entity_id).create_default_playbook(entity_id)
+    elif entity_type == FileType.TEST_PLAYBOOK:
+        pack.create_test_playbook(entity_id).create_default_test_playbook(entity_id)
+    elif entity_type == FileType.CLASSIFIER:
+        content = {'id': entity_id, 'name': entity_name, 'transformer': '', 'keyTypeMap': {}, 'type': 'classification'}
+        pack.create_classifier(entity_id, content)
+    elif entity_type == FileType.LAYOUT:
```

Comment:
Are you sure it's `layout` and not `layoutcontainer`?

Reply:
Yes. 'layoutcontainer' has no key 'kind' instead it has the 'group' key

Reply:
So, what about layoutcontainer then?

Reply:
In the UT there is no a pack depends on another pack by a layoutcontainer yet
But for the future you are right we should be able to create a layoutcontainer also
So I added that